### ISSUE TITLE
Android 7.1.1 - Xbox One A controller binding issues.

### ISSUE THREAD

**Issue by Dardashti** [libretro/RetroArch#4324]
I'm unable to bind the right analog stick on my Xbox One S in RetroArch. Android picks up the input just fine. Right stick doesn't work in games as a result.

**Comment by fr500** [comment]
Does it work in other apps?

**Comment by Dardashti** [comment]
Yes, in controller testing apps it registers as AXIS RX and AXIS RY, as
shown in the attached screenshot.

**Comment by fr500** [comment]
You forgot the screenshot... you can't attach images on the mail directly

**Comment by Dardashti** [comment]
Apologies. Here you are:
https://drive.google.com/file/d/0BwefIWS_R7K7V3RtNGl0eFBlMVU/view?usp=drivesdk

On Dec 27, 2016 4:46 PM, "Andrés" <notifications@github.com> wrote:

You forgot the screenshot... you can't attach images on the mail directly

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/libretro/RetroArch/issues/4324#issuecomment-269387105>,
or mute the thread

**Comment by fr500** [comment]
No idea... I don't have such a pad.
Do you have a xbox 360 pad to test the same via otg cable and check if the values are the same?

**Comment by Dardashti** [comment]
OTG cable?
I tried connecting the controller to my pixel xl with the USB cable and the
phone crashes and restarts. Doesn't seem to enjoy it much.

**Comment by fr500** [comment]
I think @bparker06 has a BT xbone pad

**Comment by autofire372** [comment]
The problem is actually on Android's end—it lacks an internal key mapping for the Xbox One S controller, and it doesn't play nicely as a result.

**Comment by fr500** [comment]
yeah it probably needs a KL file.
Shield ATV does have one, at least in for the BT pads and it's working alright.

**Comment by bparker06** [comment]
Right analog stick does not work for me either, just R3.

**Comment by orbea** [comment]
@Dardashti and @bparker06 Was this ever fixed?
